### ISSUE TITLE
License Choice Discussion

### ISSUE THREAD

**Issue by carinaya** [UBC-MDS/mathdistops#121]
Regarding the choice of license, we currently use the MIT license.

Personally, I believe this is a smart choice. It's straightforward, making it easy for others to use and share our work. All they need to do is acknowledge our contribution. It's also a widely recognized license, which means most people are already comfortable with it. This can make our project more accessible and inviting to users and contributors. 

What do you think my team?

You can refer to this link for guidance: https://choosealicense.com/

**Comment by sivakornchong** [comment]
I agree that at this stage of our project, we are open to making our project accessible to users and contributors. Moreover, 
since the codes are in development stage, it is better to have limited liability for usage of our codes. 

Hence, MIT license is suitable.

**Comment by czwcandy** [comment]
I also agree with MIT license. It is one of the most popular open-source licenses. This widespread acceptance can make it easier for other developers to adopt and contribute to the project. Essentially, it allows for almost unrestricted freedom as long as the original license and copyright notice are included with any substantial portions of the software.

**Comment by sandygross** [comment]
Thank you for initiating the discussion. I definitely agree with the points you made above. Our package is primarily designed as an educational resource to simplify the understanding of cumulative probability and quantiles across various distributions, as well as to aid those who require these calculations for further projects. Consequently, it is imperative that we opt for a license that imposes minimal restrictions on reuse. The MIT License grants users the freedom to use, copy, modify, merge, publish, distribute, sublicense, and sell copies of our software, with the sole condition of including the original copyright notice and the license text. I am confident that such openness will foster greater participation in our project. Synthesizing our perspectives, it's clear we are unified in our desire for our package to embody simplicity and permissiveness. Additionally, the MIT License's compatibility with numerous other licenses ensures our code can be seamlessly integrated with other software, significantly enhancing its value within the open-source ecosystem. 

Additionally, I think that GPLv3 may not be the best choice as it is copyleft license, meaning any derivative work must also be distributed under the same license terms. This can be restrictive when integrating with software under certain other licenses.
